### DISCUSSION TITLE
【odbc外表+mysql原生外表冲突导致】使用mysql外部表be奔溃

### DISCUSSION THREAD

**Discussion by action2013** [apache/doris#14118] [A - General / Q&A] [answered]
mysql版本：8.0.21
doris版本: master分支
具体操作如下：
![image](https://user-images.githubusercontent.com/7157818/200787901-360c91fa-a1ec-4a0c-9262-b082425e59c2.png)

**Comment by action2013** [comment, marked as answer]
原因已找到，使用WITH_MYSQL=1开启了原生mysql外表，与新版的odbc外表冲突了，编译参数如下：
![image](https://user-images.githubusercontent.com/7157818/200829600-e970ba2f-1c60-419a-a61a-d62f8c393817.png)

参考官方文档[Doris编译](https://doris.apache.org/zh-CN/docs/install/source-install/compilation)

参考官方文档[原生mysql外表与新版的odbc外表的说明](https://doris.apache.org/zh-CN/docs/ecosystem/external-table/odbc-of-doris)

**Reply by zy-kkk** [reply]
是的，MySQL外表与ODBC外表不能同时兼容
